packages/angular/cli/models/command-runner.ts

```diff
@@ -115,7 +115,11 @@ export async function runCommand(
 
   // if no commands were found, use `help`.
   if (commandName === undefined) {
-    commandName = 'help';
+    if (args.length === 1 && args[0] === '--version') {
```

Comment:
Do you think it would make sense to make this an argument to the `help` command?

Reply:
I think I'd keep them separate.  Their underlying purpose and implementation is sufficiently different.